src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py

```diff
@@ -466,14 +466,16 @@ def get_chart_mapping_schema(
         )
         if not os.path.exists(mappings_path):
             raise InvalidTemplateError(
-                f"ERROR: The helm package '{helm_package.name}' does not have a valid values mappings file. \
-                    The file at '{helm_package.path_to_mappings}' does not exist.\n\
-                    Please fix this and run the command again."
+                f"ERROR: The helm package '{helm_package.name}' does not have a valid"
+                " values mappings file.                     The file at"
+                f" '{helm_package.path_to_mappings}' does not exist.\n                 "
+                "   Please fix this and run the command again."
```

Comment:
Again are the spaces correct here?

Reply:
Sorry, and thanks for catching. I think that was the merge from main messing things up :-(